integration_tests/CMakeLists.txt

```
@@ -1248,6 +1248,7 @@ RUN(NAME line_continuation_01 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc wasm
 RUN(NAME line_continuation_02 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc wasm)
 RUN(NAME line_continuation_03 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc wasm)
 RUN(NAME line_continuation_04 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc wasm)
+RUN(NAME line_continuation_05 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc wasm)
```

The failure is related to wasm backend.

You can test this locally by using `lfortran integration_tests/line_continuation_05.f90 --backend=wasm`
or run the wasm backend tests using `python integration_tests/run_tests.py -j=8 -b=wasm`

Possible fix: Either fix the WASM backend or remove the wasm test here:
```suggestion
RUN(NAME line_continuation_05 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
```